feat(service): Add Viber

irfansofyana opened the pull request:
## Description

Implement Viber service to solve #114 

## Motivation and Context
- This PR is to solve #114 

## How Has This Been Tested?
- Unit Test
![image](https://user-images.githubusercontent.com/33191443/194713436-6595ca30-c03a-45b0-80e6-cd634c8457d8.png)

- End-to-end testing
![image](https://user-images.githubusercontent.com/33191443/194713794-f803ae7f-1ce5-4ddf-8afb-c2a567568bc4.png)

## Screenshots / Output (if appropriate):
- Screenshot End to end testing
The script that is used can be seen in the README with some modifications to change dummy values.
![image](https://user-images.githubusercontent.com/33191443/194713794-f803ae7f-1ce5-4ddf-8afb-c2a567568bc4.png)

- Unit Test
![image](https://user-images.githubusercontent.com/33191443/194713436-6595ca30-c03a-45b0-80e6-cd634c8457d8.png)

- All Test Results
There is a failure in `syslog` service but it's not related with this PR, maybe we can solve it in different PR?
![image](https://user-images.githubusercontent.com/33191443/194713388-170a8289-3e3c-46eb-ad8f-470363590303.png)


## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (no code change)
- [ ] Refactor (refactoring production code)
- [ ] Other <!--- (provide information) -->

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.


<!--- Credit: https://github.com/orhun/git-cliff/blob/main/.github/PULL_REQUEST_TEMPLATE.md -->

svaloumas commented on service/viber/README:
```diff
@@ -0,0 +1,75 @@
+# viber
```
```suggestion
# Viber 

[![go.dev reference](https://img.shields.io/badge/go.dev-reference-007d9c?logo=go&logoColor=white&style=flat)](https://pkg.go.dev/github.com/nikoksr/notify/service/viber)
```

irfansofyana commented on service/viber/viber_test.go:
```diff
@@ -0,0 +1,103 @@
+package viber
```
Hi @svaloumas sure! thank you for your review. I will go through your review.

irfansofyana commented:
hey @svaloumas thanks for your time on reviewing this. I've addressed your comments, please help to re-review it again, thank you! 

also @nikoksr would appreciate if you have any comment on this PR. Thank you!

nikoksr commented:
@irfansofyana I appreciate your hustle. I firmly believe @svaloumas reviews are of a consistent high quality. There won't be additional value my review could provide.

svaloumas commented:
@irfansofyana Apologies for getting back late to this! Let's just remove this sneaky line [here](https://github.com/nikoksr/notify/pull/415/files#diff-b8b704307ee6122e74f9edc773c4a733f7bad939f57ddee92da45963a839a077R38) that causes the linter failure and we're ready to go! 👍

irfansofyana commented:
@svaloumas upss sorry for that, it should be fixed now. Thanks!
